app/src/test/java/com/github/sdp/ratemyepfl/adapter/util/AdapterUtilTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.github.sdp.ratemyepfl.adapter.util
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class AdapterUtilTest {
```

Comment:
The diffCallback is already tested when running our list activities. We don't need these tests.

Reply:
It was not covered in the coverage when I made my draft. I'll let them since it tests the function independently.